src/test/fs/test_trim_caps.cc

```diff
@@ -78,8 +77,10 @@ int main(int argc, char *argv[])
 		ret = ceph_statx(cmount, ".", &stx, 0, 0);
 		assert(ret >= 0);
 
-		printf("waiting for crash\n");
-        sleep(60);
+		printf("waiting for 60 seconds to see whether will it crash\n");
+		sleep(60);
+		printf("parent exits\n");
```

Comment:
Will process with non-zero ID be parent? Shouldn't this be child?

Reply:
The `fork()` will return `0` in the child process and the child pid in the parent process, more detail please see the `man fork`:

```
RETURN VALUE
       On success, the PID of the child process is returned in the parent, and 0 is returned in the child.  On failure, -1 is returned in the parent, no child process is created, and errno is set to indicate the error.

```